frappe/desk/doctype/bulk_update/bulk_update.js

```diff
@@ -22,8 +22,16 @@ frappe.ui.form.on('Bulk Update', {
 
 					if (failed.length && !r._server_messages) {
 						frappe.throw(__('Cannot update {0}', [failed.map(f => f.bold ? f.bold(): f).join(', ')]));
+					} else {
+						frappe.msgprint({
+							title: __('Success'),
+							message: __('Updated Successfully'),
+							indicator: 'green'
+						});
 					}
+
 					frappe.hide_progress();
+					frm.save();
```

Comment:
Why saving bulk update form ? The only purpose is to update data, saving this doesn't make sense

Reply:
To maintain versions? Also 'Not Saved' is confusing for the user
![bulk_update_fix](https://user-images.githubusercontent.com/24353136/67750107-8a89e300-fa54-11e9-8a3b-379e4e7db1b9.png)